src/OpenFeature.Contrib.Providers.Flipt/README.md

```diff
@@ -0,0 +1,195 @@
+# Flipt .NET Provider
+
+An OpenFeature Provider which enables the use of the Flipt Server-Side SDK.
+
+# .NET SDK usage
+
+## Install dependencies
+
+We will first install the **flipt provider**.
+
+#### .NET CLI
+```shell
+dotnet add package OpenFeature.Contrib.Providers.Flipt
+```
+#### Package Manager
+
+```shell
+NuGet\Install-Package OpenFeature.Contrib.Providers.Flipt
+```
+#### Package Reference
+
+```xml
+<PackageReference Include="OpenFeature.Contrib.Providers.Flipt" />
+```
+#### Paket CLI
+
+```shell
+paket add OpenFeature.Contrib.Providers.Flagd
+```
+
+
+## Using the Flipt Provider with the OpenFeature SDK
+
+Using the Flip provider assumes that you have installed and running Flipt service. Instructions on how to do this can be found in the [official documentation](https://docs.flipt.io/self-hosted/overview)
+
+The following example shows how to use the Flipt provider with the OpenFeature SDK.
+
+*appsettings.json*
+```json
+{
+    "OpenFeature": {
+        "Provider": {
+            "Flipt": {
+                "ServiceUri": "http://localhost:9000",
+                "Namespace": "default",
+                "TargetingKey": "UserId",
+                "RequestIdKey": "RequestId",
+                "UseBooleanEvaluation": false
+            }
+        }
+    }
+}
+```
+
+*Program.cs*
+
+```csharp
+using OpenFeature.Contrib.Providers.Flipt;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Get provider config from configuration
+var config = builder.Configuration.GetSection("OpenFeature:Provider:Flipt").Get<FliptProviderConfiguration>();
+
+/* Or create config directly:
+var config = new FliptProviderConfiguration
+{
+    ServiceUri = new Uri("http://localhost:9000"),
```

Comment:
I think we'll want to support authentication as well, likely with some helpers that can add either clientToken authentication (https://docs.flipt.io/authentication/using-tokens) or JWT authentication (https://docs.flipt.io/authentication/using-jwts)

The only difference really is the value of the `Authorization` header that is sent to Flipt:
- Client Token: `Authentication: Bearer {token}`
- JWT: `Authentication: JWT {token}`

Reply:
also support for our authentication methods still needs to be added @dmitryrogov